metaci/build/autoscaling.py

```diff
@@ -180,6 +198,25 @@ def scale_max(self, url, headers, worker_type, limit):
 
         return requests.patch(url, json={"quantity": target_workers}, headers=headers)
 
+    def one_off_build(self, build_config):
+        config_args, no_lock = build_config[0:-1], build_config[-1]
+        if no_lock:
+            config_args.extend(("--no-lock"))
+        command = " ".join(
+            [
+                "python",
+                "./manage.py",
+                "metaci_rqworker",
+                *config_args,
+            ]
+        )
+        url = f"{self.base_url}/dynos"
+        json = {"command": command, "time_to_live": "86400"}
+
+        resp = requests.post(url, json=json, headers=self.headers)
+        resp.raise_for_status()
```

Comment:
Can we do this in a way that makes the response from the Platform API visible to us? i.e. it's not terribly helpful to know that there was a 500 error without seeing the response body